aea/cli/generate.py

```diff
@@ -49,18 +47,19 @@ def _generate_item(ctx: Context, item_type, item_name, specification_path):
     existing_id_list = getattr(ctx.agent_config, "{}s".format(item_type))
     existing_item_list = [public_id.name for public_id in existing_id_list]
 
-    # loader = getattr(ctx, "{}_loader".format(item_type))
-    # default_config_filename = globals()["DEFAULT_{}_CONFIG_FILE".format(item_type.upper())]
-
     item_type_plural = item_type + "s"
 
-    protocol_template = ProtocolTemplate(specification_path)
+    try:
```

Comment:
Please merge this and the following try, except pair.

Reply:
Done